plugins/languages/markdown/glow.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "glow";
+  originalName = "glow.nvim";
+  defaultPackage = pkgs.vimPlugins.glow-nvim;
+
+  maintainers = [ lib.maintainers.getchoo ];
+
+  settingsOptions = {
+    glow_path = helpers.defaultNullOpts.mkStr (helpers.mkRaw "vim.fn.exepath('glow')") ''
+      Path to `glow` binary
+
+      Consider using `glowPackage` instead
+    '';
+
+    install_path = helpers.defaultNullOpts.mkStr "~/.local/bin" ''
+      Path for installing `glow` binary if one is not found
+
+      Consider using `glowPackage` instead
+    '';
+
+    border = helpers.defaultNullOpts.mkEnumFirstDefault [
+      "shadow"
+      "none"
+      "double"
+      "rounded"
+      "solid"
+      "single"
+    ] "Style of the floating window's border";
+
+    style = helpers.defaultNullOpts.mkEnum [
+      "dark"
+      "light"
+    ] (lib.literalExpression "vim.opt.background") "Glow style";
```

Comment:
```suggestion
    ] (helpers.mkRaw "vim.opt.background") "Glow style";
```

Reply:
that seems to trigger [this](https://github.com/nix-community/nixvim/blob/97fa47376b73d774b081c8e5dd54fcfd0ad278cb/lib/options.nix#L223) assertion. any good way around it?

Reply:
Looks like you've found a bug! 🪲

The assertion doesn't handle raw types correctly.

Just double checking, the type is definitely an enum right?

Reply:
> Just double checking, the type is definitely an enum right?

yep! you can find the definition of it upstream [here](https://github.com/ellisonleao/glow.nvim/blob/238070a686c1da3bccccf1079700eb4b5e19aea4/lua/glow.lua#L14) and `vim.opt.background` being used as a default [here](https://github.com/ellisonleao/glow.nvim/blob/238070a686c1da3bccccf1079700eb4b5e19aea4/lua/glow.lua#L32). this seems accurate to what's described as recognized values of `background` in [neovim's documentation](https://neovim.io/doc/user/options.html#'background')

Reply:
#1837 should resolve the issue